Source/UnitTests/VideoCommon/VertexLoaderTest.cpp

```diff
@@ -106,6 +106,15 @@ class VertexLoaderTest : public testing::Test
   std::unique_ptr<VertexLoaderBase> m_loader;
 };
 
+template <typename T>
```

Comment:
Would it make sense to move this function into MathUtil? (MathUtil::SafeFloatToInt?)